src/gdext/surface/properties.nim

```diff
@@ -252,63 +355,115 @@ template `@export_exp_easing`*[T: SomeUserClass; S: SomeProperty](
   register_property(typedesc T, name, typedesc S, getter, setter,
     hint= propertyHintExpEasing,
     hint_string= @extra.mapIt($it).join(","))
+template gdexport_exp_easing*(iden: propertyinfo.SomeFloat, extra: varargs[ExpEasingArgument], alias: static[alias] = noAlias) =
+  register_property_iden(`iden`, `alias`, hint= propertyHintExpEasing, hint_string= "attenuation")
 
-template `@export_multiline`*[T: SomeUserClass; S: SomeProperty](
+template gdexport_multiline*[T: SomeUserClass; S: SomeString](
     name;
     getter: proc(self: T): S;
     setter: proc(self: T; value: S)): untyped =
   register_property(typedesc T, name, typedesc S, getter, setter,
     hint= propertyHintMultilineText)
+template gdexport_multiline*(iden: SomeString, alias: static[alias] = noAlias) = 
+  register_property_iden(`iden`, `alias`, hint= propertyHintMultilineText)
 
-template `@export_node_path`*[T: SomeUserClass; S: SomeProperty](
+template gdexport_node_path*[T: SomeUserClass; S: SomeProperty](
     name;
     getter: proc(self: T): S;
     setter: proc(self: T; value: S);
     validTypes: varargs[string]): untyped =
   register_property(typedesc T, name, typedesc S, getter, setter,
-    hint= propertyHintNodePathValidTypes,
+    hint= propertyHintNodeType,
     hint_string= @validTypes.join(","))
-
-macro `@export_node_path`*[T: SomeUserClass; S: SomeProperty](
+macro gdexport_node_path*[T: SomeUserClass; S: SomeProperty](
     name;
     getter: proc(self: T): S;
     setter: proc(self: T; value: S);
     validTypes: varargs[typedesc[Node]]): untyped =
-  result = bindSym"@export_node_path".newCall(name, getter, setter)
+  result = bindSym"gdexport_node_path".newCall(name, getter, setter)
   for valid in validTypes:
     result.add bindSym"$".newCall bindSym"className".newCall valid
+template gdexport_node_path*(iden: SomeUserClass, alias: static[alias] = noAlias) = 
+    gdexport(`iden`, `alias`)
```

Comment:
This seems still mock-up.

Reply:
I kinda forgot about that one. I fixed it in the latest commit